lib/dialects/sqlite/query.js

```diff
@@ -66,10 +64,10 @@ class Query extends AbstractQuery {
   }
 
   _handleQueryResponse(metaData, columnTypes, benchmark, queryBegin, err, results) {
-    debug(`executed(${this.connection.uuid || 'default'}) : ${this.sql}`);
+    debug(`executed(${this.connection.uuid || 'default'}) : ${this.sql} with parameters ${logger.inspect(this.options.bind)}`);
```

Comment:
`util.inspect` is rather expensive function to call, we should really not call it unless we really need to in production code.
I think a better approach would be to redirect all `debug` calls to the logger itself and then check if the debug message should be emitted before doing the expensive calls.
This would look like `logger.debug({ action: string/*executed etc.*/, sql, bound?, connId?, ... })`
This would allow us to dedupe a fair chunk of logging code too!

If you want I could take a stab at this.

Reply:
I see the problem. `logger.inspect` is always called even if debug is not enabled, which drags performance. I like the solution of using `logger.debug`. And yes, it'd be great if you could help me with this.

Reply:
II have been thinking about the logging issues. The concerns raised at #10630 are difficult to solve without debug.

I didn't know that `debug` had [formatters](https://www.npmjs.com/package/debug#formatters) and more importantly, that they are [configurable](https://www.npmjs.com/package/debug#environment-variables).

For instance:

```js
debug('executed(%s) : %s with parameters %O', this.connection.uuid || 'default', this.sql, this.options.bind);
```

I think they can satisfy the requirements of @sushantdhiman (configurability), @SimonSchick (performance) and this pull request (verbose query params). What do you think?

Reply:
Thanks for your patch, I've merged it with this pull request.